docs/topics/auth/passwords.txt

```diff
@@ -173,10 +216,32 @@ default PBKDF2 algorithm:
             'django.contrib.auth.hashers.CryptPasswordHasher',
         ]
 
-
 That's it -- now your Django install will use more iterations when it
 stores passwords using PBKDF2.
 
+Argon2
+~~~~~~
+
+Argon2 has three attributes that can be customized:
+
+#. ``time_cost`` controls the number of iterations within the hash
+#. ``memory_cost`` controls the size of memory that must be used during the
+   computation of the hash
+#. ``parallelism`` controls how many CPUs the computation of the hash can be
+   parallelized on.
+
+The default values of these attributes are probably fine for you.  If you
```

Comment:
Should we say what the defaults are? Should we consider increasing the defaults for each major release of Django like we do for PBKDF2? https://docs.djangoproject.com/en/dev/internals/howto-release-django/#new-stable-branch-tasks
Please use 1 space between sentences.